content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,114 @@
+---
+title: InfluxDB Cloud limits and adjustable quotas
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Adjustable quotas and limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account quotas per organization and hard global system limits.
+
+{{% warn %}}
+All __rates__ (data-in (writes), queries (reads), and deletes) are accrued within a fixed five-minute window. Once a rate is exceeded an error response is returned until the current five-minute window resets.
+{{% /warn %}}
+
+Review adjustable service quotas and global limits to plan for your bandwidth needs:
+
+- [Adjustable service quotas](#adjustable-service-quotas)
+- [Global limits](#global-limits)
+- [UI Error Messages](#ui-error-messages)
+<!-- - [API Error Responses](#api-error-responses) -->
+
+## Adjustable service quotas
+
+To reduce the chance of unexpected charges and protect the service for all users, InfluxDB Cloud has adjustable service quotas applied per account.
+
+_To request higher service quotas, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Free Plan
+
+- **Data-in**: Rate of 5 MB per 5 minutes (average of 17 kb/s)
+  - Uncompressed bytes of normalized [line protocol](/influxdb/cloud/reference/syntax/line-protocol/)
+- **Read**: Rate of 300 MB per 5 minutes (average of 1000 kb/s)
+  - Bytes in HTTP in response payload
+- **Cardinality**: 10k series (see [cardinality](/influxdb/cloud/reference/glossary/#series-cardinality))
+- **Available resources**:
+  - 2 buckets (excluding `_monitoring` and `_tasks` buckets)
+  - 5 dashboards
+  - 5 tasks
+- **Alerts**:
+  - 2 checks
+  - 2 notification rules
+  - Unlimited Slack notification endpoints
+- **Storage**: 30 days of data retention (see [retention period](/influxdb/cloud/reference/glossary/#retention-period))
+
+{{% note %}}
```

Comment:
This note applies to the Usage-based Plan and Annual plans...but it feels like it is hanging next to the Free plan...which has a max 30 day retention.

Reply:
thank you, @timhallinflux --moved this text to **Storage** under Usage-based Plan.